.github/workflows/static_analysis.yaml

```diff
@@ -74,3 +74,21 @@ jobs:
 
       - name: Run Linter
         run: "yarn run lint:style"
+
+  analyse_dead_code:
+    name: "Analyse Dead Code"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v3
+        with:
+          cache: 'yarn'
+
+      - name: Install Deps
+        run: "scripts/ci/layered.sh"
+
+      - name: Dead Code Analysis
+        run: |
+          cd element-web 
+          yarn run analyse:unused-exports
```

Comment:
does this script exit with a non-zero code when there is dead code?

Reply:
I do believe it does